hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/AbstractBackendStoreProvider.java

```diff
@@ -131,10 +133,15 @@ public void clear() throws BackendException {
     }
 
     @Override
-    public void truncate() {
+    public void truncate(HugeGraph graph) {
         this.checkOpened();
+        HugeConfig config = (HugeConfig) graph.configuration();
+        String systemStoreName = config.get(CoreOptions.STORE_SYSTEM);
         for (BackendStore store : this.stores.values()) {
-            store.truncate();
+            // Don't truncate system store
+            if (store.store().equals(systemStoreName)) {
+                store.truncate();
+            }
```

Comment:
it's more reasonable to truncate systemStore + graphStore, can we move meta table from systemStore to schemaStore?